dev/dags/example_simple_http_operator_task.yml

```diff
@@ -0,0 +1,35 @@
+default:
+  default_args:
+    catchup: false,
+    start_date: 2025-03-20
+
+http_operator_example_dag:
+  default_args:
+    owner: "@owner"
+  description: "this is an HttpOperator dag"
+  schedule_interval: "0 3 * * *"
+  tags: ['http']
+  render_template_as_native_obj: True
+  tasks:
+    send_request_json:
+      operator: airflow.providers.http.operators.http.SimpleHttpOperator
+      http_conn_id: "example_host"
+      method: "POST"
+      endpoint: "/run_test"
+      data:
+        data: "fake_data"
+        format: "json"
```

Comment:
do we need this format key here, how are we using this in the implementation?

Reply:
It is a just example and I choose not clear key for json. 
It can be like 
```
name: "fake_name"
lastname: "fake_lastname"
```
I used key format:json for defining plain text and json format in tests